src/library_egl.js

```diff
@@ -23,6 +23,10 @@ var LibraryEGL = {
     currentContext: 0 /* EGL_NO_CONTEXT */,
     currentReadSurface: 0 /* EGL_NO_SURFACE */,
     currentDrawSurface: 0 /* EGL_NO_SURFACE */,
+    alpha: false,
+    depth: true,
+    stencil: true,
+    antialias: true,
 
```

Comment:
Are these defaults set according to EGL's defaults, or WebGL, or just based on the previous defaults we had here? Please add a comment on this, as e.g. the eglChooseConfig docs mention depth and stencil should be false, so we should document the difference there.

Reply:
Right, I just avoided changing the previous behavior, I'll add a comment.
Would you rather we follow EGL's defaults though?

Reply:
Let's leave that for a later PR - I'm not sure what's the best thing.